spec/rspec/rails/configuration_spec.rb

```diff
@@ -256,6 +256,31 @@ def self.application; end
       expect(group.mailer_class).to be(a_mailer_class)
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
+
+    describe 'cleans test mailbox between each example in all rspec-rails example' do
+      class BaseMailer < ActionMailer::Base
+        default from: 'from@example.com'
+
+        def welcome(to:)
+          mail(to: to, subject: 'subject', body: render(inline: "Hello", layout: false))
+        end
+      end
+      before do
+        ActionMailer::Base.delivery_method = :test
+      end
+
+      it 'send to email@' do
+        BaseMailer.welcome(to: 'email@example.com').deliver_now
+
+        expect(ActionMailer::Base.deliveries.map(&:to).flatten.sort).to eq(['email@example.com'])
+      end
+
+      it 'send to email_2@' do
+        BaseMailer.welcome(to: 'email_2@example.com').deliver_now
+
+        expect(ActionMailer::Base.deliveries.map(&:to).flatten.sort).to eq(['email_2@example.com'])
+      end
```

Comment:
take it this fails without the change? if so see above then good to go!

Reply:
Thanks for the quick review.

I need both test to have one that fail. Maybe I could rewrite them to be more explicit but I thought it would make the test more complicated to understand. Like having explicit `example.run`. I don't know.

Reply:
I'm just checking that they fail without the change to `lib/rspec/rails/configuration.rb`

Reply:
yes they are. :)